### PR TITLE
Adding VBF

### PR DESCRIPTION
**Pull request by dprim7**
Added VBF category for multiclass BDT

### CODE REVIEW

**Review by rkansal47 — CHANGES_REQUESTED** [review]
Could you remove the `.ipynb_checkpoints` folders from your commits? And the `.swp` ones.

File: src/HH4b/sensitivity_study/plotting.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import uproot
```
Comment (rkansal47) [inline src/HH4b/sensitivity_study/plotting.py:1]: What is this?

File: src/HH4b/postprocessing/postprocessing.py [hunk 2]
```diff
@@ -108,8 +122,74 @@ class Syst:
 )
 def postprocess(years):
     # TODO: set this as a yaml file
+<<<<<<< HEAD
     dirs = {"/eos/uscms/store/user/cmantill/bbbb/skimmer/Oct2/": samples}
 
+=======
```
Comment (rkansal47) [inline src/HH4b/postprocessing/postprocessing.py:128]: This file looks like it wasn't merged properly (has merge conflict markers)

**Review by rkansal47 — CHANGES_REQUESTED** [review]

**Comment by rkansal47** [comment]
Should we close this in favour of your newer one?

**Comment by cmantill** [comment]
Closing since Raghav integrated changes in a separate PR

**Pull request closed without merging**